libs/designsystem/src/lib/components/button/button.component.scss

```diff
@@ -44,11 +45,35 @@ $button-width: (
   }
 }
 
+// Mixin parameters must be a named color variant. CSS color values won't work.
+// See libs/core/src/scss/themes/_colors.scss
```

Comment:
This is just me being nitpicky; feel free to leave as is - but consider not referring directly to a file path in comments. 

If we ever change the location this comment will be invalid and we might not remember to update the comment 😄

Reply:
To throw an additional tool on top of the mountain of tools that constitutes a _modern_ (put into that word what you wish) frontend stack - a tool that makes it possible to reference files & objects would be kinda cool; failing if they do not exist anymore. 

// Mixin parameters must be a named color variant. CSS color values won't work.
// See: libs/core/src/scss/themes/_colors.scss$[file-ref]

Reply:
We will without a doubt most certainly forget to update the comment if we move the file to a different location 😁.

I'll remove line 49:

```
// See libs/core/src/scss/themes/_colors.scss
```